src/foam/nanos/menu/MenuBar.js

```diff
@@ -84,6 +84,10 @@ foam.CLASS({
         return false;
       }
 
+      if ( this.selected === menu.id && this.selected.parent != 'settings' ) {
```

Comment:
Why is this required? Add comment to explain.

Reply:
I have actually changed it because this wasnt persisting in every case, I used window.location.hash.includes('#set') so that if the user navigates to any settings pages the previous menu items wont stay underlined. With the latest commit the menu items work as they should and only show as selected if the user is on that page and has not navigated to a settings page.

Reply:
I think this makes sense as if any new menu items are added to the settings menu they will include the hash of "#set" in them